knowledge-base/treeview-rename-node.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: How to Rename a TreeView Node
-description: How to edit the name (text) of a treeview node
+description: Learn how to edit the name (text) of a Treeview node and explore the example that demonstrates a possible approach with an ItemTemplate. 
 type: how-to
-page_title: How to rename a TreeView node?
+page_title: How to Rename a TreeView Node?
 slug: treeview-kb-rename-node
 position: 
-tags: edit, rename, tree, treeview, note, item
-ticketid: 1525532
+tags: edit, rename, tree, treeview, node, item
```

Comment:
Tags should make sense outside of context. "Node" and "item" are too general. But you can use template if you like.

```suggestion
tags: telerik, treeview, tree, edit, template
```